info/guide/guide-chapter-6.md

```diff
@@ -428,6 +428,22 @@ fun SomeClass.deleteAllSpaces() {
 }
 ```
 
+#### <a name="r6.2.4"></a> 6.2.4 Don't use property length with operation - 1
+You should not use property length with operation - 1, you can change this to lastIndex
```

Comment:
Instead of using `.length - 1` use built-in Kotlin method `lastIndex`, it makes the code more readable.

Reply:
Done